Procfile

```
@@ -2,7 +2,7 @@ release: bin/release
 web: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-prod.conf.py warehouse.wsgi:application
 web-api: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-prod.conf.py warehouse.wsgi:application
 web-uploads: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-uploads.conf.py warehouse.wsgi:application
-worker: bin/start-worker celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
+worker: bin/start-worker celery -A warehouse worker $EXTRA_CELERY_ARGS -Q default -l info --max-tasks-per-child 1024
```

Should this be something like
```suggestion
worker: bin/start-worker celery -A warehouse worker --concurrency $CELERY_CONCURRENCY -Q default -l info --max-tasks-per-child 1024
```
instead?

---

Downside here is we _must_ set a value of $CELERY_CONCURRENCY, but its an option.

---

If no value is set, we'll get:

```
worker-1        | Error: Invalid value for '-c' / '--concurrency': '--beat' is not a valid integer.
```

---

lemme see if bash expansion works in procfiles lol.

---

It does. reworking!